PostHog/Utils/UIWindow+.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  UIWindow+.swift
+//  PostHog
+//
+//  Created by Yiannis Josephides on 03/12/2024.
+//
+
+#if os(iOS) || os(tvOS)
+    import UIKit
```

Comment:
do we need to import `Foundation`? i recall having an issue once if not doing this despite compiling

Reply:
found the issue https://github.com/PostHog/posthog-ios/pull/194/files specifically URLSessionExtension
If `UIKit` knows about String, Bool and type(of...), should be fine, your call here.